Automattic/woocommerce-payments
# Send CVC confirmation to server for fraud prevention

Fixes #4121

#### Changes proposed in this Pull Request

* Send a new CVC confirmation flag to the server to process it when the transaction is being created.









#### Testing instructions

* As we still are implementing it on the Platform Checkout and need it for the WCPay 4.0 launch, just test the normal user flow (add something to cart and place order), everything should work normally.





-------------------


**Post merge**
